### PR TITLE
Add Pancakeswap v3 trades

### PR DESCRIPTION
**Pull request by hsrvc**
Brief comments on the purpose of your changes:

**Add Pancakeswap v3 trades**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
* [x ] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @hsrvc for the great work!

**Comment by hsrvc** [comment]
Thanks @Hosuke 🙏. Anything I need to on my side to fix the dbt seed check?

**Comment by hsrvc** [comment]
Hi @Hosuke any chance we can fix dbt-test issue soon? much appreciated 🙏

**Comment by hsrvc** [comment]
Hi @Hosuke just added Pancakeswap CAKE token on Ethereum and fixed the wrong one. Sorry it kind of breaks the test again

File: models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 1]
```diff
@@ -3238,7 +3238,7 @@ FROM (VALUES
         ('0x26fed67c97776baf1bdd9f40097686a904b25ef9', 'INVIS', 18),
         ('0x26ff6d16549a00ba8b36ce3159b5277e6e798d18', 'CHIHUA', 18),
         ('0x2700bb35cc0965de65f24934976d54a8291ceaa4', 'CAT', 8),
-        ('0x27015e5df08a390565259195781d5217e65f3654', 'CAKE', 18),
+        ('0x27015e5df08a390565259195781d5217e65f3654', 'Piece of (CAKE)', 18),
```
Comment (Hosuke) [inline models/tokens/ethereum/tokens_ethereum_erc20.sql:3241]: https://etherscan.io/token/0x27015e5df08a390565259195781d5217e65f3654

**Review by Hosuke — APPROVED** [review]
LGTM. for the token addresses update.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**